rollup-interface/src/node/rpc/mod.rs

```diff
@@ -104,110 +129,165 @@ impl Default for QueryMode {
     }
 }
 
+/// The body of a response to a JSON-RPC request for a particular slot.
 #[derive(Debug, PartialEq, Eq, Clone, Serialize, Deserialize)]
 pub struct SlotResponse<B, Tx> {
+    /// The slot number
     pub number: u64,
+    /// The hex encoded slot hash
```

Comment:
Should we mention in other places that hash suppose to be hex encoded?

Reply:
Good idea, I see you flagged one that I missed - I'll do a quick check for others